app/models/invoice.rb

```diff
@@ -534,18 +534,6 @@ def self.opened_by_buyer(buyer)
            .reorder('period DESC, created_at DESC').first
   end
 
-  # TODO: are those needed without provider/buyer scope?
-  # The month should be a YYYY-MM formated string.
-  def self.find_by_month(month)
-    by_month(month).first
-  end
-
-  # TODO: are those needed without provider/buyer scope?
-  def self.find_by_month!(month)
-    find_by_month(month) ||
-      raise(ActiveRecord::RecordNotFound, "Couldn't find #{name} by month=#{month}")
-  end
-
```

Comment:
Why are you removing these?

Reply:
Because they are not used. There were already comments mentioning that we might not need these, because they are not scoped by provider/buyer. And it is reasonable.
So I checked, and I couldn't find their usage outside of the tests.